suse_migration_services/units/prepare.py

```diff
@@ -163,12 +177,12 @@ def main():
             zypp_plugins_services, '/usr/lib/zypp/plugins/services'
         )
         if os.path.exists(cloud_register_metadata):
```

Comment:
With the change above this changes to `if cloud_register_metadata and os.path.exists(cloud_register_metadata):`

Reply:
this could be more streamlined if I move it all to line  165 for example:  Any objections?
```
        with open(hosts_setup, 'r', encoding="utf-8") as fp:
            if 'susecloud.net' in fp.read():
                try:
                    cloud_register_metadata = \
                        get_regionsrv_client_file_location(root_path)
                    log.info('Bind mounting /var/cache/cloudregister from {0}'.format(
                        cloud_register_metadata)
                    )
                    Path.create('/var/cache/cloudregister')
                    Command.run(
                        [
                            'mount', '--bind', cloud_register_metadata,
                            '/var/cache/cloudregister'
                        ]
                    )
                    update_smt_cache = '/usr/sbin/updatesmtcache'
                    if os.path.isfile(update_smt_cache):
                        log.info('Updating SMT cache')
                        Command.run(
                            [update_smt_cache]
                        )
                except DistMigrationZypperMetaDataException as issue:
                    log.error('Failed locating regionsrv-client cache files: {0}'.format(
                        issue)
                    )
                    raise
 ```

Reply:
I will update it per your changes. f you want me to switch to the above that is fine or we can keep it as is.